File: BackEnd-Siukat/models/verifikasi.go
package models

type Verifikasi struct {
	IDVerifikasi    int    `gorm:"primaryKey;column:id_verifikasi" json:"id_verifikasi"`
	NoPeserta       int    `gorm:"column:no_peserta" json:"no_peserta"`
	VerAkademik     string `gorm:"column:ver_akademik;type:enum('lolos','tidak_lolos','belum_verifikasi')" json:"ver_akademik"`
	KetAkademik     string `gorm:"column:ket_akademik;type:varchar(255)" json:"ket_akademik"`
	VerBidikMisi    string `gorm:"column:ver_bidik_misi;type:enum('lolos','tidak_lolos','belum_verifikasi')" json:"ver_bidik_misi"`
	KetBidikMisi    string `gorm:"column:ket_bidik_misi;type:varchar(255)" json:"ket_bidik_misi"`
	VerKeterampilan string `gorm:"column:ver_keterampilan;type:enum('lolos','tidak_lolos','belum_verifikasi')" json:"ver_keterampilan"`
	KetKeterampilan string `gorm:"column:ket_keterampilan;type:varchar(255)" json:"ket_keterampilan"`
	VerKjmu         string `gorm:"column:ver_kjmu;type:enum('lolos','tidak_lolos','belum_verifikasi')" json:"ver_kjmu"`
	KetKjmu         string `gorm:"column:ket_kjmu;type:varchar(255)" json:"ket_kjmu"`
	VerKipk         string `gorm:"column:ver_kipk;type:enum('lolos','tidak_lolos','belum_verifikasi')" json:"ver_kipk"`
	KetKipk         string `gorm:"column:ket_kipk;type:varchar(255)" json:"ket_kipk"`
}

func (Verifikasi) TableName() string {
	return "tb_verifikasi"
}

File: Be-siukat-new/siukat-service-dev_snmptn_2022/models/verifikasi.js
var Sequelize = require("sequelize");
var db = require("../config/database");

const Verifikasi = db.define(
  "verifikasi",
  {
    id_verifikasi: {
      type: Sequelize.INTEGER,
      primaryKey: true,
    },
    no_peserta: {
      type: Sequelize.INTEGER,
    },
    ver_akademik: {
      type: Sequelize.ENUM("lolos", "tidak_lolos", "belum_verifikasi"),
    },
    ket_akademik: {
      type: Sequelize.STRING,
    },
    ver_bidik_misi: {
      type: Sequelize.ENUM("lolos", "tidak_lolos", "belum_verifikasi"),
    },
    ket_bidik_misi: {
      type: Sequelize.STRING,
    },
    ver_keterampilan: {
      type: Sequelize.ENUM("lolos", "tidak_lolos", "belum_verifikasi"),
    },
    ket_keterampilan: {
      type: Sequelize.STRING,
    },
    ver_kjmu: {
      type: Sequelize.ENUM("lolos", "tidak_lolos", "belum_verifikasi"),
    },
    ket_kjmu: {
      type: Sequelize.STRING,
    },
    ver_kipk: {
      type: Sequelize.ENUM("lolos", "tidak_lolos", "belum_verifikasi"),
    },
    ket_kipk: {
      type: Sequelize.STRING,
    },
  },
  {
    // don't add the timestamp attributes (updatedAt, createdAt)
    timestamps: false,

    // don't delete database entries but set the newly added attribute deletedAt
    // to the current date (when deletion was done). paranoid will only work if
    // timestamps are enabled
    paranoid: true,

    // don't use camelcase for automatically added attributes but underscore style
    // so updatedAt will be updated_at
    underscored: true,

    // disable the modification of tablenames; By default, sequelize will automatically
    // transform all passed model names (first parameter of define) into plural.
    // if you don't want that, set the following
    freezeTableName: true,

    // define the table's name
    tableName: "tb_verifikasi",
  }
);
module.exports = Verifikasi;
